.github/linters/.yaml-lint.yml

```diff
@@ -6,10 +6,6 @@ rules:
   brackets:
     max-spaces-inside: 1
   document-start: disable
-  indentation:
-    level: warning
```

Comment:
It's the key change of this PR, reenabling the default styling rules for identation.